src/viam/examples/modules/example_mlmodelservice_tflite.cpp

```diff
@@ -744,7 +744,7 @@ int serve(const std::string& socket_path) try {
         vsdk::MLModelService::static_api(),
 
         // Declare a model triple for this service.
-        vsdk::Model{"viam", "mlmodelservice", "tflite"},
+        vsdk::Model{"viam", "mlmodelservice", "example_mlmodelservice_tflite"},
```

Comment:
This is a more substantive change. I hadn't really paid much attention to the name I'd picked here, but I needed to reference it in the generated config for the robot (see `example_audio_classification_client.cpp`). It felt wrong to landgrab `viam::mlmodelservice::tflite`. So I uglified the name to make sure it identified itself as an example.